src/Tools/Source/CompilerGeneratorTools/Source/CSharpSyntaxGenerator/AbstractFileWriter.cs

```diff
@@ -75,6 +75,12 @@ protected void WriteLine(string msg)
             _needIndent = true; //need an indent after each line break
         }
 
+        protected void WriteLineWithoutIndent(string msg)
```

Comment:
This naming is just confusing. The name is `WriteLineWithoutIndent`, and yet is specifically sets `_needIndent`.

Reply:
The current line is written without indent. The next line is not. Open to alternate names if you have a suggestion.